### PR TITLE
Dgupta/check docker issue

### PR DESCRIPTION
**Pull request by guptadivya1501**

### CODE REVIEW

File: src/ansys/dpf/core/server_factory.py [hunk 1]
```diff
@@ -155,6 +155,23 @@ def docker_run_cmd_command(self, docker_server_port: int, local_port: int) -> st
         mounted_volumes_args = "-v " + " -v ".join(
             key + ":" + val for key, val in self.mounted_volumes.items()
         )
+
+        if os.name == "nt":
+            updated_volume_args = ""
+            for i in mounted_volumes_args.split(" "):
+                if ":" in i:
+                    i = "/mnt/" + (
```
[the remark this answers is not in the document]
Comment (PProfizi) [reply src/ansys/dpf/core/server_factory.py:163]: https://www.scivision.dev/python-detect-wsl/

[the remark this answers is not in the document]
Comment (PProfizi) [reply src/ansys/dpf/core/server_factory.py:163]: It might be `.release` or `.version`

**Review by anslpa — APPROVED** [review]

**Review by PProfizi — APPROVED** [review]

**Comment by PProfizi** [comment]
hi @guptadivya1501 is this PR still relevant or can we close it as stale?

**Comment by guptadivya1501** [comment]
> hi @guptadivya1501 is this PR still relevant or can we close it as stale?

Hi @PProfizi , I want it to be open for some more days.

**Comment by PProfizi** [comment]
Hi @guptadivya1501 do you think we can close this PR as stale?

**Pull request closed without merging**